ProcessLib/Output.cpp

```diff
@@ -0,0 +1,146 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include<cassert>
+#include<fstream>
+#include<vector>
+
+#include <logog/include/logog.hpp>
+
+#include "BaseLib/FileTools.h"
+#include "Output.h"
+
+namespace
+{
+//! Determines if there should be output at the given \c timestep.
+template<typename CountsSteps>
+bool doOutput(std::size_t timestep, CountsSteps const& repeats_each_steps)
```

Comment:
i would name it `needOutput()`. `doOutput` sounds something is written into a file